lib/paths/local.go

```diff
@@ -999,4 +1026,82 @@ func (st *Local) ReadSnapVanillaProof(ctx context.Context, sr storiface.SectorRe
 	return out, nil
 }
 
+func (st *Local) supraPoRepVanillaProof(src storiface.SectorPaths, sr storiface.SectorRef, sealed, unsealed cid.Cid, ticket abi.SealRandomness, seed abi.InteractiveSealRandomness) ([]byte, error) {
```

Comment:
please describe source of the 'from' and 'to' format for the unit test author.

Reply:
Which 'from' and 'to'?